ci/properties/tencent.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Build and Deploy to TKE",
```

Comment:
Can you change the name here (and in the workflow file too) to just be the platform, such as `TKE` or `Tencent Cloud` or `Tencent Kubernetes Engine`

Reply:
done